src/tools/dev/scripts/bv_support/bv_zlib.sh

```diff
@@ -57,6 +57,14 @@ function bv_zlib_host_profile
     fi
 }
 
+function bv_zlib_initialize_vars
+{
+    export VISIT_ZLIB_DIR=${VISIT_ZLIB_DIR:-"$VISITDIR/zlib/${ZLIB_VERSION}/${VISITARCH}"}
```

Comment:
This is for the case where zlib is supported via an explicit build of it. What about the case where system zlib is used? What gets exporte for VISIT_ZLIB_DIR?

Reply:
Nothing should be exported, I will make the export conditional.

Reply:
> I will make the export conditional.

Did you alread commit that?

Reply:
Yes.  Did you notice the 'Outdated' label above.  You can see the changes made in the 'Files changed' view.

Reply:
Yes, I did. I followed the link to the file and then selected from `...` in upper right, `view file` and did not see it. My mistake however as `view file` from that screen takes me to the specific commit hash on that file and not the head of the branch. I just looked at the head and see what I expect there now. Sorry.